README.md

```diff
@@ -12,15 +12,15 @@ for complex large scale application problems.  This toolkit was
 built from scripts open sourced from [RoundingWell.com](http://roundingwell.com).
 
 ## Documentation
-  [StateClass](./docs/state-class.md) - `Marionette.Object` with a `Backbone.Model` for keeping state.
+  [App](./docs/app.md) - An extension of `Marionette.Object`. Its purpose is to provide an object with a `initialize`/`start`/`stop`/`destroy` lifecycle.
 
-  [Component](./docs/component.md) - Reusable `StateClass` with `View` management boilerplate
+  [Component](./docs/component.md) - Provides a consistent interface for which to package state-view-logic.
 
-  [AbstractApp](./docs/abstract-app.md) - Object including a start/stop lifecycle and running event registry.
+  [StateMixin](./docs/mixings/state.md) - `Marionette.Object` with a `Backbone.Model` for keeping state.
```

Comment:
This isn't a Marionette.Object though.. it's just an object for mixing into other things.